lib/util/file.js

```diff
@@ -6,24 +6,30 @@ import { resolve } from 'path';
  */
 
 /**
- * Returns the content for the given file URI
+ * Returns all files with a directories and it's sub-directories
```

Comment:
Should be "Returns all files within a directory, including those within sub-directories " or something like that